dashboard/app/views/levels/_admin.html.haml

```diff
@@ -42,6 +42,12 @@
                   - @level.properties['pages'].each do |page|
                     - page['levels'].each do |level_name|
                       %li= link_to level_name, level_path(Level.find_by_name(level_name))
+            - if project_template_level_name = @level.properties['project_template_level_name']
+              - project_template_level = Level.find_by_name(project_template_level_name)
```

Comment:
It isn't immediately obvious to me whether we want to gate the `li` creation on the level existing. Failing fast and hard seems like a reasonable route.